test/filter.js

```diff
@@ -16,6 +16,13 @@ test('With a where option with multiple statements', t => {
   run('trees', options, 1, t)
 })
 
+test('With a where option with multiple statements with appended 1=1', t => {
+  const options = {
+    where: "Genus like '%Quercus%' AND Street_Name = 'CLAREMONT' AND House_Number < 600 AND Trunk_Diameter = 9 AND 1=1"
+  }
+  run('trees', options, 1, t)
```

Comment:
Its the same test as the previous: 
`test('With a where option with multiple statements', t => {
  const options = {
    where: "Genus like '%Quercus%' AND Street_Name = 'CLAREMONT' AND House_Number < 600 AND Trunk_Diameter = 9"
  }
  run('trees', options, 1, t)
})`

but it just has AND 1=1 appended.  1=1 being the default query clause that feature server uses. 
 For some reason AGOL likes to throw that on the end of a query when filtering a layer from time to time even when there are other fields in the query.